addons/web/static/src/views/fields/properties/card_properties_field.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
-    <t t-name="web.KanbanPropertiesField">
+    <t t-name="web.CardPropertiesField">
```

Comment:
shouldn't we rename `view_in_kanban` into `view_in_cards` in the definition now that it's also used for calendar ?
(it's possible to do a migration script and to update the keys in the definition using `jsonb_set`)

Reply:
Didn't know it was possible.
I added an upgrade script for knowledge as it is the only module that is currently using the option